tests/start_services.sh

```diff
@@ -29,7 +29,6 @@ fi
 
 # restart services (down + up) checking for ports availability
 $COMPOSE_BASE --profile '*' down -v --remove-orphans
-rm -rf $TEST_DIR
```

Comment:
This is useful; however will pollute the dir and it would be really hard to find the most recent run if you forgot to do `rm tests/tmp` manually. Can we just create a dedicated `tmp` dir for each run, for instance using some environment variable with a counter?

Reply:
I don't see how creating dedicated directories would avoid polluting the dir. One can always manually delete the directory in case it gets too big. Moreover each test logs its directory so I don't see the case where it would be difficult to find the most recent run (also the file manager should help in that, since directories have a creation timestamp).

Reply:
I am telling from my experience: when multiple tests run it is newly impossible to find which test created which dir.

Reply:
When debugging something a user usually doesn't run multiple tests at once, so I can't see this as a problem. Anyway I think this can be addressed later, if needed

Reply:
I was not talking about an abstract user, I was talking about myself. And this thing would save me a lot of time. Ok, I will work on it myself.